auth/authController.js

```diff
@@ -0,0 +1,158 @@
+const jwt = require("jsonwebtoken");
+const passport = require("passport");
+const User = require("../models/schemas/user.js");
+const models = require("../models/usersFunc.js");
+require("dotenv").config();
+const secretWord = process.env.SECRET;
+
+const auth = (req, res, next) => {
+  passport.authenticate("jwt", { session: false }, (err, user) => {
+    if (!user || err) {
```

Comment:
Tu trzeba dodać jeszcze sprawdzanie czy token w req jest taki sam jak user.token Aktualnie jeśli token w user jest null to funkcja przepuszcza jak by ktoś był zalogowany i current się wykrzacza bo token jest nullem